3-Authorization-II/2-call-api-b2c/AppCreationScripts/AppCreationScripts.md

```diff
@@ -0,0 +1,148 @@
+# Registering sample apps with the Microsoft identity platform and updating configuration files using PowerShell
```

Comment:
Guess we can remove this file since there are no registration scripts in this sampple